css/document.css

```diff
@@ -9,6 +9,12 @@
 html {
   font: 10.5pt/1.15 'Noto Serif', serif;
   text-align: justify;
+  text-align-last: left;
+}
+
+/* if a paragraph has been broken up on two pages, justify the text on the last line of the first page */
+.paragraph [data-split-original] {
```

Comment:
Does the space has a meaning or is it equivalent to `.paragraph[data-split-original]` ?

Reply:
It makes a difference in CSS AFAIK.

Reply:
I'm not a CSS expert but according to Stackoverflow (the source of all truth, wisdom and lies):

> If you do, the space is treated as a descendant combinator instead, with the universal selector implied on the attribute selector and anything that may follow it.

So if I understand correctly, it means that it would match:

```html
<div class="paragraph">
<p data-split-original="">
</p>
</div>
```

But not:

```html
<div class="paragraph" data-split-original="">
<p>
</p>
</div>
```

Correct?

Reply:
Right. The CSS that is in the PR worked for me. I would have to dig deeper and see more edge cases, for now I would like to leave it as is.